packages/flutter/lib/src/cupertino/multicolumn_picker.dart

```diff
@@ -0,0 +1,39 @@
+// Copyright 2017 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:flutter/widgets.dart';
+
+import 'picker.dart';
+
+/// An iOS picker with multiple columns.
+/// Consists of a list of CupertinoPickers, and this class will display them
+/// sequencely on a row. User can define the flex of each child in the row by
+/// passing a list of int values for flex.
+/// Customization of CupertinoPicker is implicitly defined in each picker.
+class MultiColumnCupertinoPicker extends StatelessWidget {
+  /// Create a multi column picker from single column pickers.
+  const MultiColumnCupertinoPicker({
+    Key key,
+    @required this.children,
+    this.flexes,
+  }): super(key: key);
+
+  /// List containing flexes of children.
+  final List<int> flexes;
+
+  /// List containing single column pickers.
+  final List<CupertinoPicker> children;
```

Comment:
We generally try to avoid having widgets that has only other specific widget children. 

I think we don't need this class and we can just document in CupertinoWidgets that they can use a Row with MainAxisAlignment to achieve the desired visual effect.

Reply:
Initially Dave and I think about implementing a generic multicolumn picker, so I created the class, but yeah, I will discuss with Dave about this.

Reply:
I have removed the class.